calculator/calculator.js

```diff
@@ -0,0 +1,16 @@
+const calculator = {
+  add(a, b) {
+    return a + b;
+  },
```

Comment:
Very nice! Since you are returning just one expression, another option of which I'm sure you are aware is an arrow function: ` add: (a, b) => a + b`.